rel-eng/build-packages-for-obs.sh

```diff
@@ -140,6 +140,26 @@ while read PKG_NAME PKG_VER PKG_DIR; do
   ${VERBOSE:+cat "$T_LOG"}
 
   eval $(awk '/^Wrote:.*src.rpm/{srpm=$2}/^Wrote:.*.changes/{changes=$2}END{ printf "SRPM=\"%s\"\n",srpm; printf "CHANGES=\"%s\"\n",changes; }' "$T_LOG")
+  EXTRA_CHANGELOGS=0
+  if [ "$(head -n1 ${CHANGES}|grep '^- ')" == "" ]; then
+    PREVIOUS_CHANGELOGS=0
+  else
+    PREVIOUS_CHANGELOGS=1
+  fi
+  FILELIST="$(ls ${GIT_DIR}/${PKG_DIR}${PKG_NAME}.changes.* 2> /dev/null; true)"
```

Comment:
Here we send errors to /dev/null and we always return 0, so we don't break the script.

If no separate changelogs are found, then the section is skipped.